### ISSUE TITLE
Summation errors due to CDR-related energy

### ISSUE THREAD

**Issue by katarkow** [pik-piam/remind2#552]
Due to the assumption that emissions from using `fegas` to produce heat for DAC are captured, there are summation errors in the reporting. It happens because  `vm_emiTeMkt` accounts for burning all that `fegas` used for DAC and also synfuels from that captured carbon, so we account for that carbon twice. I thought of the following solutions two solutions:

(1) Updating the `vm_emiTeMkt` in REMIND, so discounting these emissions * share CCU or
(2) Discounting the CCU from gas for DAC in remind2, (a) either from `vm_emiTeMkt` and `vm_emiAllMkt` directly where they’re declared or (b) everywhere where they’re used, so `Emi|CO2`, `Emi|CO2|+|Energy`, `Emi|GHG`, …. (potentially painful to remember about this when a new reporting variable uses either of the REMIND variables).

What are your thoughts?

I tried (1)  [here](https://github.com/remindmodel/remind/pull/1589), I thought it would be the cleanest. However, there's a small accounting error in the reporting from using `sm_eps` to handle cases in which there’s no captured CO2 in a given (t, regi): `share = ccs / (captured + sm_eps)`.

**Comment by katarkow** [comment]
I tried (1)  [here](https://github.com/remindmodel/remind/pull/1589), I thought it would be the cleanest. However, there's a small accounting error in the reporting from using `sm_eps` to handle cases in which there’s no captured CO2 in a given (t, regi): `share = ccs / (captured + sm_eps)`. Anybody has a better idea to handle this? :)

(2a) Is here: #553

**Comment by LaviniaBaumstark** [comment]
is this still an issue in latest runs: /p/projects/remind/runs/REMIND_2024_09_24/remind/output ?

**Comment by orichters** [comment]
`/p/projects/remind/runs/REMIND_2024_09_24/remind/output/SSP2-PkBudg650_2024-09-24_13.59.41/log.txt` still has this error:
```
Carbon Sequestration|CCS >                                                    + Carbon Management|Storage >
   + Carbon Sequestration|CCS|Biomass                                            + Carbon Management|Storage|+|Biomass|Pe2Se + Carbon Management|Storage|Industry Energy|+|Biomass
   + Carbon Sequestration|CCS|Fossil                                             + Carbon Management|Storage|+|Fossil|Pe2Se + Carbon Management|Storage|Industry Energy|+|Fossil
   + Carbon Sequestration|Direct Air Capture                                     - Emi|CO2|CDR|DACCS
   + Carbon Sequestration|CCS|Industrial Processes                               + Carbon Management|Storage|+|Industry Process
The child sum differs by -100% to -0.244% from the parent, absolute difference up to 758 Mt CO2/yr.
```

**Comment by tabeado** [comment]
@orichters the big deviation in your example comes from a missing change to the summation checks when plastics-CCS was added: "Carbon Management|Storage" currently contains all the variables in the image attached. The variable "Carbon Management|Storage|+|Waste|Plastics Incineration" reaches the value of 757.8 Mt globally in 2090 in that run. 

There are still some other inconsistencies in the accounting related to the this issue here (which is being discussed also in 2 other issues), but these are minor (total always less that <0.5 Mt CO2/yr), as you can see e.g. in this run: /p/projects/remind/runs/REMIND_2024_09_24/remind/output/SSP2-EU21-PkBudg650_2024-09-24_13.21.33/log.txt 

I would suggest the following:
1) adjust the Carbon Sequestration|CCS check by adding Plastic waste incineration 
2) close this issue for a better overview as it is also being discussed in [issue293](https://github.com/remindmodel/development_issues/issues/293) and [issue344](https://github.com/remindmodel/development_issues/issues/344) 

![image](https://github.com/user-attachments/assets/288549ea-12f1-4faa-a677-06e86d2e259b)

**Comment by orichters** [comment]
Thanks, Tabea! As we cannot adjust the AR6 reporting as such and there is no plausible category for Waste incineration, I would suggest we just start ignoring the AR6 issues altogether. Thanks!

**Comment by fschreyer** [comment]
I agree that we should close this issue and #293. For new summation errors, please create new issues.
